Read.txt

```diff
@@ -1 +1,4 @@
-add this line please.
\ No newline at end of file
+add this line please.
+
+
+add new line
```

Comment:
Much better if you add dot(.) on this line

Reply:
Done.